apps/application-status/pages/sprints/[team].vue

```diff
@@ -0,0 +1,110 @@
+<template>
+  <div>
+    <ul>
+      <li v-for="team in curTeams" :key="team.id">
+        <NuxtLink :to="team.path">
+          {{ team.title }}
+        </NuxtLink>
+      </li>
+    </ul>
+    <button @click="changeEndCursor">
+      Next
+    </button>
+    <button @click="changeStartCursor">
+      Prev
+    </button>
+    <p> {{ teamName }} </p>
+    <pre>
+            {{ display }}
+        </pre>
+  </div>
+</template>
+
+<script lang="ts">
+import getSprints from '../../helper/getSprints'
+import teams from '../../data/TeamsData'
+import { PageInfo, TeamSprints } from '../../interface/interfaces'
+import { getReleases } from '../../composables/getReleases'
+export default {
+  data () {
+    const route = useRoute()
+    const teamName = route.params.team
+    const display: TeamSprints = {
+      totalCount: 0,
+      pageInfo: {
+        endCursor: '',
+        hasNextPage: false,
+        hasPreviousPage: false,
+        startCursor: ''
+      },
+      sprints: []
+    }
+    const pageInfo: PageInfo = {
+      startCursor: '',
+      endCursor: '',
+      hasNextPage: false,
+      hasPreviousPage: false
+    }
+    const prev: string[] = ['']
+    const curTeams = teams
+    const boardId = ''
+    const keyWord = ''
+    return {
+      teamName,
+      pageInfo,
+      display,
+      prev,
+      curTeams,
+      boardId,
+      keyWord
+    }
+  },
+  beforeMount () {
+    this.displayData()
+  },
+
+  methods: {
+    async displayData () {
+      if (this.boardId === '' && this.keyWord === '') {
+        if (this.teamName === 'entity') {
+          this.boardId = this.curTeams[0].id
+          this.keyWord = this.curTeams[0].keyWord
+        } else if (this.teamName === 'nameteamspace') {
+          this.boardId = this.curTeams[1].id
+          this.keyWord = this.curTeams[1].keyWord
+        } else if (this.teamName === 'assets') {
+          this.boardId = this.curTeams[2].id
+          this.keyWord = this.curTeams[2].keyWord
+        } else if (this.teamName === 'relationships') {
+          this.boardId = this.curTeams[3].id
+          this.keyWord = this.curTeams[3].keyWord
+        } else if (this.teamName === 'btr') {
+          this.boardId = this.curTeams[4].id
+          this.keyWord = this.curTeams[4].keyWord
+        }
+      }
+      const releases = await getReleases(this.boardId)
+      const data = await getSprints(this.boardId, this.pageInfo.startCursor, this.pageInfo.endCursor, releases, this.keyWord)
+      this.display = data
+      this.pageInfo = this.display.pageInfo
+    },
+    changeStartCursor () {
```

Comment:
not very clear what functions changeStartCursor and changeEndCursor are supposed to do (e.g. why is startCursor reset in both?) Usually when this type of convoluted logic is added to code - tests would be helpful to show what the intended results are. Were you planning to add a testing framework?

Reply:
Oh the start and end cursors is for pagination, ZenhubAPI provides start and end cursor for us to fetch specific numbers of data in 1 page, in this case is it'll fetch 5 sprints per page, and then start and end cursor is when moving to next page, it will fetch the next 5 sprints instead of fetching all at once

Reply:
Oh I just tested 2 per page because the new boards don't have a lot of sprints to test with, fetching 2 at a time will result in more pages to test with navigating back and forth, but after verifying everything is fine, I will switch to 10 per page